app/api/files/jsRoutes.js

```diff
@@ -46,13 +47,21 @@ export default app => {
     '/api/public',
     multer().any(),
     captchaAuthorization(),
-    (req, _res, next) => {
-      req.body = JSON.parse(req.body.entity);
-      return next();
-    },
-    validation.validateRequest(saveSchema),
+    validation.validateRequest(
+      Joi.object().keys({
+        entity: saveSchema,
+        email: Joi.object().keys({
+          to: Joi.string(),
+          sender: Joi.string(),
+          text: Joi.string(),
+          html: Joi.string(),
+          subject: Joi.string(),
+        }),
+      })
```

Comment:
validateRequest already supports using json schema, please use that for new validations so we can progressively remove joi validations, example in the same file.  

https://github.com/huridocs/uwazi/blob/5a9d015835141b16187708f3f86d64f800a01970/app/api/files/jsRoutes.js#L151-L174